packages/profiler-app/cli.ts

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+// Server entry point for the profiler app
+// This script processes support bundles at runtime, then starts the appropriate server
+//
+// Similar to viz-bundle.py, this script:
+// - Extracts profile and config files from a support bundle
+// - Generates dataflow graph from SQL using the compiler
+// - Saves files to data/ directory
+// - Starts the web server (Vite dev or static file server)
+//
+// Usage:
+//   Development mode:
+//     bun run dev                                    # Start without bundle processing
+//     BUNDLE=/path/to/bundle.zip bun run dev         # Process bundle then start dev server
+//     BUNDLE=/path/to/bundle.zip VERBOSE=1 bun run dev  # With verbose output
+//
+//   Production mode (after running 'bun run build'):
+//     bun run start                                  # Serve built files
+//     BUNDLE=/path/to/bundle.zip bun run start       # Process bundle then serve
+//     BUNDLE=/path/to/bundle.zip BUNDLE_NAME=my-profile bun run start  # Custom name
+//
+//   Direct execution:
+//     node cli.ts                                    # Auto-detects dev/production
+//     BUNDLE=/path/to/bundle.zip node cli.ts         # With bundle processing
+
+import { BundleProcessor, getBundleFromEnv } from './src/bundleProcessor.js';
+import { spawn } from 'child_process';
+import { fileURLToPath } from 'url';
+import { dirname, join } from 'path';
+import { existsSync } from 'fs';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+async function main() {
+    // Process bundle if BUNDLE env var is set
+    const bundleOptions = getBundleFromEnv();
+
+    if (bundleOptions) {
+        try {
+            console.log('[Profiler] Processing support bundle...');
+            const processor = new BundleProcessor(bundleOptions);
+            const outputName = await processor.process();
+            console.log(`[Profiler] Bundle processed successfully: ${outputName}`);
```

Comment:
I would say not "processed", but probably "data extracted"; same in the catch block